miageWebApp/src/app/cours/cours.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { Cours } from '../model/cours';
+import { CoursService } from '../services/cours.service';
+import { ThrowStmt } from '@angular/compiler';
+
+@Component({
+  selector: 'app-cours',
+  templateUrl: './cours.component.html',
+  styleUrls: ['./cours.component.scss']
+})
+
+export class CoursComponent implements OnInit {
+  myClasses: Cours[] = [];
+  addClassForm: FormGroup;
+
+  constructor(private coursService: CoursService, private formBuilder: FormBuilder) {
+    this.addClassForm = this.formBuilder.group({
+      titleInput: '',
+      periodInput: '',
+      nbHourInput: '',
+      teacherInput: '',
+      descriptionInput: ''
+    });
```

Comment:
```suggestion
this.classForm = this.formBuilder.group({
      title: '',
      period: '',
      nbHour: '',
      teacher: '',
      description: ''
    });
```

Reply:
I don't undrestand when the name should be easily understand and  don't it

Reply:
Actually, here the names are not correct because in the formBuilder.group({...}) you don't declare inputs but form controls. So maybe titleFormControl, periodFormControl, nbHourFormControl... but not inputs